package.json

```diff
@@ -24,19 +24,19 @@
     "flow-parser": "0.38.0",
     "get-stdin": "5.0.1",
     "glob": "7.1.1",
-    "jest-validate": "18.2.0",
+    "jest-validate": "^19.0.0",
```

Comment:
yarn seems to add `^` to versions in package.json by default, but this project uses fixed versions.